bubuku/env_provider.py

```diff
@@ -35,46 +36,59 @@ def create_env_provider(config: Config):
 
 class AmazonEnvProvider(EnvProvider):
     def __init__(self, config: Config):
-        self.document = None
         self.aws_addr = '169.254.169.254'
         self.config = config
+        self.ip_address = None
 
     def _get_document(self) -> dict:
-        if not self.document:
-            try:
-                self.document = requests.get(
-                    'http://{}/latest/dynamic/instance-identity/document'.format(self.aws_addr),
-                    timeout=5).json()
-                _LOG.info("Amazon specific information loaded from AWS: {}".format(
-                    json.dumps(self.document, indent=2)))
-            except Exception as ex:
```

Comment:
Because for "Amazon" class it's really exception when it's not possible to connect to AWS